source/docs/zero-to-robot/step-2/index.rst

```diff
@@ -1,4 +1,15 @@
-Updating and Configuring Pneumatics Control Module and Power Distribution Panel
-===============================================================================
+Step 2: Installing Dependencies
+===============================
 
 .. warning:: The CAN Web Dashboard plugin is no longer supported by the roboRIO Web Dashboard. To configure CTRE CAN devices such as the PCM and PDP, use `CTRE Phoenix Tuner <https://phoenix-documentation.readthedocs.io/en/latest/ch03_PrimerPhoenixSoft.html#what-is-phoenix-tuner>`_
```

Comment:
Why is this warning here, and not in the `configuring CAN devices` section/page?

Reply:
The page was a stub and isn't really needed. I'm considering adding a 3rd party page that has the mention.